frontend/src/app/modules/home/projects/edit/edit.component.ts

```diff
@@ -0,0 +1,144 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { BaseComponent } from '@core/components/base/base.component';
+import { AuthenticationService } from '@core/services/authentication.service';
+import { ConfirmWindowService } from '@core/services/confirm-window.service';
+import { ProjectService } from '@core/services/project.service';
+import { SpinnerService } from '@core/services/spinner.service';
+import { ToastNotificationService } from '@core/services/toast-notification.service';
+import { AlertSettings } from '@shared/models/alert-settings/alert-settings';
+import { Organization } from '@shared/models/organization/organization';
+import { Platform } from '@shared/models/platforms/platform';
+import { Project } from '@shared/models/projects/project';
+import { UpdateProject } from '@shared/models/projects/update-project';
+import { User } from '@shared/models/user/user';
+import { MenuItem, PrimeIcons } from 'primeng/api';
+import { DialogService } from 'primeng/dynamicdialog';
+import { switchMap } from 'rxjs/operators';
+import { Data } from '../data';
+
+@Component({
+    selector: 'app-edit',
+    templateUrl: './edit.component.html',
+    styleUrls: ['./edit.component.sass'],
+    providers: [Data, DialogService]
+})
+export class EditComponent extends BaseComponent implements OnInit {
+    user: User;
+    organization: Organization;
+    platforms = {
+        platformTabItems: [] as MenuItem[],
+        activePlatformTabItem: {} as MenuItem,
+        viewPlatformCards: [] as Platform[],
+        platformCards: [] as Platform[]
+    };
+    alertSetting = {} as AlertSettings;
+    editForm: FormGroup = new FormGroup({});
+    project: Project;
+    updateProject = {} as UpdateProject;
+    id: string;
+    activeTabIndex: number = 0;
+    dropPlatform: Platform[];
```

Comment:
Remove unused variables here, for example updateProject.

Reply:
Fix